codegen/src/test/resources/q3/Q3.json

```diff
@@ -0,0 +1,120 @@
+{
+  "join_structure": [
+    {
+      "PrimaryRelation": "Orders",
+      "ForeignRelation": "Lineitem"
+    },
+    {
+      "PrimaryRelation": "Orders",
+      "ForeignRelation": "Customer"
+    }
+  ],
+  "LineitemProcessFunction": {
+    "name": "Q3lineitem",
+    "relation": "lineitem",
+    "this_key": [
+      "orderkey"
+    ],
+    "next_key": [
+      "orderkey"
+    ],
+    "child_nodes": 1,
+    "is_Root": true,
+    "is_Last": true,
+    "rename_attribute": null,
+    "select_conditions": {
+      "operator": "&&",
+      "value1": {
+        "left_field": {
+          "type": "attribute",
+          "name": "l_shipdate"
+        },
+        "right_field": {
+          "type": "constant",
+          "var_type": "Date",
+          "value": "1995-03-15"
+        },
+        "operator": ">"
+      }
+    }
+  },
+  "OrdersProcessFunction": {
+    "name": "Q3Orders",
+    "relation": "orders",
+    "this_key": [
+      "custkey"
+    ],
+    "next_key": [
+      "orderkey"
+    ],
+    "child_nodes": 1,
+    "is_Root": true,
+    "is_Last": true,
+    "rename_attribute": null,
+    "select_conditions": {
+      "operator": "&&",
+      "value1": {
+        "left_field": {
+          "type": "attribute",
+          "name": "orderdate"
```

Comment:
This one should be o_orderdate? Please check if the attribute name matches our schema in the codegen.